parlai/tasks/multiwoz_v22/agents.py

```diff
@@ -393,5 +605,5 @@ class SingleApiSchemaAgent(MultiwozV22Parser, tod_agents.TodSingleApiSchemaAgent
     pass
 
 
-class DefaultTeacher(SystemTeacher):
```

Comment:
would be better to keep the default to SystemTeacher as is.

Reply:
Fixed this.